Add support for multiple linked_orderpositions on reusable media (#5666)
pretix/pretix: doc/api/resources/reusablemedia.rst

type                                  string                     Type of medium, e.g. ``"barcode"``, ``"nfc_uid"`` or ``"nfc_mf0aes"``.
organizer                             string                     Organizer slug of the organizer who "owns" this medium.
identifier                            string                     Unique identifier of the medium. The format depends on the ``type``.

> secret                                string                     Secret to identify the medium (or ``null``)

Reviewer: That appears confusing. When comparing this with OrderPosition or GiftCard together with this description, one could think that `secret` is the thing that is printed on the medium and scanned, such as the barcode content. However, that is not what this `secret` is for. If I recall correctly, this is more like a "token to add this to a customer account". 

At the minimum, the description here should make that clearer, but ideally we find another name than "secret" that makes the purpose more clear. Could very well be that we need other kinds of secret things on this model in the future. Maybe `claim_token` or `claim_secret`?

label                                 string                     Label to identify the medium, usually something human readable (or ``null``)
active                                boolean                    Whether this medium may be used.
created                               datetime                   Date of creation
updated                               datetime                   Date of last modification
expires                               datetime                   Expiry date (or ``null``)
customer                              string                     Identifier of a customer account this medium belongs to.
linked_orderpositions                 list of integer            Internal IDs of tickets this medium is linked to.

> linked_orderposition                  integer                    Internal ID of a ticket this medium is linked to. (**deprecated**, do not use, will be removed)

Reviewer: For consistency with other API docs:

```suggestion
linked_orderposition                  integer                    **DEPRECATED.** ID of one the ticket the medium is linked to, if it is linked to only one ticket. ``null``, if the medium is linked to none or multiple tickets.
```

type                                  string                     Type of medium, e.g. ``"barcode"``, ``"nfc_uid"`` or ``"nfc_mf0aes"``.
organizer                             string                     Organizer slug of the organizer who "owns" this medium.
identifier                            string                     Unique identifier of the medium. The format depends on the ``type``.
claim_token                           string                     Secret token to claim ownership of the medium (or ``null``)
label                                 string                     Label to identify the medium, usually something human readable (or ``null``)
active                                boolean                    Whether this medium may be used.
created                               datetime                   Date of creation
updated                               datetime                   Date of last modification
expires                               datetime                   Expiry date (or ``null``)
customer                              string                     Identifier of a customer account this medium belongs to.

> linked_orderpositions                 list of integer            Internal IDs of tickets this medium is linked to.

Reviewer: ```suggestion
linked_orderpositions                 list of integers           Internal IDs of tickets this medium is linked to.
```

- ``nfc_uid``
- ``nfc_mf0aes``

> .. versionchanged:: 2025.11

Reviewer: ```suggestion
.. versionchanged:: 2026.5
```

   :query string customer: Only show media linked to the given customer.
   :query string created_since: Only show media created since a given date.
   :query string updated_since: Only show media updated since a given date.
   :query integer linked_orderpositions: Only show media linked to the given tickets. Note: you can pass multiple ticket IDs by passing

>                          ``linked_orderpositions`` multiple times. Any medium matching any linked orderposition will be returned.

Reviewer: ```suggestion
                                         ``linked_orderpositions`` multiple times. Any medium matching any linked orderposition will be returned.
```
